src/commands/run.rs

```diff
@@ -170,6 +176,13 @@ impl RunCommand {
 
         let preopen_sockets = self.compute_preopen_sockets()?;
 
+        // Validate coredump-on-trap argument
+        if let Some(coredump_path) = self.coredump_on_trap.as_ref() {
+            if coredump_path.contains("%") {
+                bail!("The coredump-on-trap path does not support patterns yet.")
```

Comment:
As a minor nit stylistically our error messages start with lowercase letters (including the "Core dumped at ..." message below)